src/qiskit_sphinx_theme/theme/qiskit-sphinx-theme/theme.conf

```diff
@@ -1,11 +1,6 @@
 [theme]
 inherit = furo
-
-# Our stylesheet takes precedence.
-stylesheet =
-  styles/furo.css,
-  styles/furo-extensions.css,
```

Comment:
We don't need to explicitly register `furo-extensions.css` apparently. It already gets registered by Furo, thanks to us inheriting Furo.